packages/engine/Source/Scene/I3SGeometry.js

```diff
@@ -267,6 +269,18 @@ I3SGeometry.prototype.getClosestPointIndexOnTriangle = function (px, py, pz) {
   };
 };
 
+function convertColorFactor(factor) {
+  const convertedFactor = [];
+  for (let i = 0; i < factor.length; i++) {
```

Comment:
Based on our coding guide, this should be.
 
```suggestion
  const length = factor.length;
  for (let i = 0; i < length; i++) {
```

Reply:
Done.